cobbler/remote.py

```diff
@@ -3911,6 +3972,16 @@ def get_tftp_file(
         self.check_access(token, "get_tftp_file")
         return self.api.get_tftp_file(path, offset, size)
 
+    def begin_transaction(self, token: str) -> bool:
```

Comment:
I would change the names to `transaction_begin`, `transaction_commit`, etc.

Reply:
done